openassessment/assessment/test/test_staff.py

```diff
@@ -210,23 +218,68 @@ def test_create_assessment_does_not_block(self, after_type, after_assess):
 
         # Verify both assessment and workflow report correct score
         self.assertEqual(staff_assessment["points_earned"], OPTIONS_SELECTED_DICT[staff_score]["expected_points"])
-        workflow = workflow_api.get_workflow_for_submission(tim_sub["uuid"], self.STEP_REQUIREMENTS)
+        workflow = workflow_api.get_workflow_for_submission(tim_sub["uuid"], requirements)
         self.assertEqual(workflow["score"]["points_earned"], OPTIONS_SELECTED_DICT[staff_score]["expected_points"])
 
         # Now, non-force asses with a 'most' value
         # This was selected to match the value that the ai test will set
         unscored_assessment = OPTIONS_SELECTED_DICT["most"]
         assessment = after_assess(tim_sub["uuid"], tim["student_id"], unscored_assessment["options"])
-        # and update workflow with new scores
-        requirements = self.STEP_REQUIREMENTS
-        if after_type == 'peer':
-            requirements = {"peer": {"must_grade": 0, "must_be_graded_by": 1}}
 
         # Verify both assessment and workflow report correct score (workflow should report previous value)
         self.assertEqual(assessment["points_earned"], unscored_assessment["expected_points"])
         workflow = workflow_api.get_workflow_for_submission(tim_sub["uuid"], requirements)
         self.assertEqual(workflow["score"]["points_earned"], OPTIONS_SELECTED_DICT[staff_score]["expected_points"])
 
+    def test_provisionally_done(self):
```

Comment:
Actually nevermind @cahrens - this particular test already includes a user making a peer assessment after having a staff assessment present. So yes, the bug should be fixed by this PR.